index/index_test.go

```diff
@@ -34,12 +34,13 @@ func TestWriteJSONNonleafRows(t *testing.T) {
 		"testing.leaf",
 		"testing.nonleaf.",
 		"testing.leaf.node",
+		"testing.\"broken\".node",
 	}
 	metrics, err := writeRows(rows)
 	if err != nil {
 		t.Fatalf("Error during transform or unmarshal: %s", err)
 	}
-	if len(metrics) != 2 {
+	if len(metrics) != 3 {
 		t.Fatalf("Wrong metrics slice length = %d: %s", len(metrics), metrics)
 	}
 	if metrics[0] != "testing.leaf" || metrics[1] != "testing.leaf.node" {
```

Comment:
May be add test for metrics[2] ?

Reply:
@msaf1980 what do you mean? There is an additional metric in the test, so there are not 3, not 2.

Reply:
Ah understood, updated.